types/src/schemas/stoatConfigSchema.json

```diff
@@ -129,6 +132,30 @@
         }
       }
     },
+    "metric_plugin_map": {
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/metric_plugin"
+      }
+    },
+    "metric_plugin": {
+      "type": "object",
+      "required": ["filename"],
+      "properties": {
+        "metadata": {
+          "type": "object",
+          "additionalProperties": true
+        },
+        "unit": {
+          "type": "string",
+          "description": "The unit will be shown in the y-axis title in the metric chart."
+        },
+        "filename": {
```

Comment:
Should this be `path` for consistency?

Reply:
The `path` can be either a directory or a file. So I guess `filename` is more specific?

Currently the workflow dispatch plugin also has a `filename` field. So this is consistent with that plugin.